runtime/lua/vim/ui.lua

```diff
@@ -55,6 +55,90 @@ function M.select(items, opts, on_choice)
   end
 end
 
+--- Prompts the user to pick one or multiple items from a collection of entries
```

Comment:
I know the docs for `ui.select` uses the verb "pick", but IMO we should stick to a single verb ("select" in this case).

```suggestion
--- Prompts the user to select one or more items from a collection of entries
```